tests/kernel/semaphore/semaphore/src/main.c

```diff
@@ -553,8 +613,83 @@ void test_sem_take_multiple(void)
 
 	signal_count = k_sem_count_get(&low_prio_sem);
 	zassert_true(signal_count == 1U,
-		     "low priority thread didn't get executed");
+			"Low priority thread don't get the sem");
+}
+
+/**
+ * @brief Test the max value a semaphore can be given and taken
+ * @details
+ * - Reset an initialized semaphore's count to zero.
+ * - Give the semaphore by a thread and verify the semaphore's count whether as
+ *   expected.
+ * - Verify the max count a semaphore can reach.
+ * - Take the semaphore by a thread and verify the semaphore's count whether
```

Comment:
whether --> is as expected

Reply:
there are a few other occurrences throughout - please adjust.

Reply:
Replaced everywhere.